tests/plugins/test_entry_point.py

```diff
@@ -72,10 +72,10 @@ def select(group, name):  # pylint: disable=unused-argument
 
 @pytest.mark.parametrize(
     'eps, name, exception', (
-        ((EP(name='ep', group='gr', value='x'),), None, None),
```

Comment:
why is that change necessary?

Reply:
Together with the change below on line 95, the changes should have identical behavior. So I guess this is fine.